src/Context/authContext.js

```diff
@@ -121,6 +122,7 @@ const AuthProvider = ({ children }) => {
   };
 
   const logout = () => {
+    localStorage.clear();
```

Comment:
Good that you are clearing all the items at once.

Reply:
okay will reduce the padding and use `console.error`.